src/PostgREST/Config.hs

```diff
@@ -283,7 +283,7 @@ readOptions = do
           |# root-spec = "stored_proc_name"
           |
           |## content types to produce raw output
-          |# raw-media-types=["image/png","image/jpg"]
+          |# raw-media-types="image/png,image/jpg"
```

Comment:
nitpick: Add a space between the types.`"image/png, image/jpg"`

Reply:
Ok